### PR TITLE
community: add oracle autonomous database doc loader integration

### PR DESCRIPTION
**Pull request by xsai9101**
Thank you for contributing to LangChain!

- [ ] **PR title**: "package: description"
  - Where "package" is whichever of langchain, community, core, experimental, etc. is being modified. Use "docs: ..." for purely docs changes, "templates: ..." for template changes, "infra: ..." for CI changes.
  - Example: "community: add foobar LLM"

- [ ] **PR message**: ***Delete this entire checklist*** and replace with
    - **Description:** Adding oracle autonomous database document loader integration. This will allow users to connect to oracle autonomous database through connection string or TNS configuration.
    https://www.oracle.com/autonomous-database/
    - **Issue:** None
    - **Dependencies:** oracledb python package 
    https://pypi.org/project/oracledb/
    - **Twitter handle:** if your PR gets announced, and you'd like a mention, we'll gladly shout you out!

- [ ] **Add tests and docs**: If you're adding a new integration, please include
  1. a test for the integration, preferably unit tests that do not rely on network access,
  2. an example notebook showing its use. It lives in `docs/docs/integrations` directory.
  Unit test and doc are added.

- [ ] **Lint and test**: Run `make format`, `make lint` and `make test` from the root of the package(s) you've modified. See contribution guidelines for more: https://python.langchain.com/docs/contributing/

Additional guidelines:
- Make sure optional dependencies are imported within a function.
- Please do not add dependencies to pyproject.toml files (even optional ones) unless they are required for unit tests.
- Most PRs should not touch more than one package.
- Changes should be backwards compatible.
- If you are adding something to community, do not re-import it in langchain.

If no one reviews your PR within a few days, please @-mention one of baskaryan, efriis, eyurtsev, hwchase17.

### CODE REVIEW

File: docs/docs/integrations/document_loaders/oracleautonomousdatabase.ipynb
Comment (baskaryan) [inline docs/docs/integrations/document_loaders/oracleautonomousdatabase.ipynb:1]: could we update this file to have same name as module? ie oracleadb_loader.ipynb

File: libs/community/langchain_community/document_loaders/oracleadb_loader.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    Columns are written into the `page_content` and 'metadata' in
+    constructor is written into 'metadata' of document,
+    by default, the 'metadata' is None.
+    """
+
+    def __init__(
+        self,
+        query: str,
+        user: str,
+        password: str,
```
Comment (baskaryan) [inline libs/community/langchain_community/document_loaders/oracleadb_loader.py:25]: nit could we add `*,` between password and schema, to enforce that schema onwards should be specified as named args

**Review by baskaryan — APPROVED** [review]
two comments otherwise lgtm

File: libs/community/langchain_community/document_loaders/oracleadb_loader.py (hunk 1 above)
Comment (xsai9101) [reply libs/community/langchain_community/document_loaders/oracleadb_loader.py:25]: Sure, I will add *.

File: docs/docs/integrations/document_loaders/oracleautonomousdatabase.ipynb
Comment (xsai9101) [reply docs/docs/integrations/document_loaders/oracleautonomousdatabase.ipynb:1]: Sure, will change the name.

**Pull request merged**